share/completions/gping.fish

```diff
@@ -0,0 +1,10 @@
+# Completion for gping
```

Comment:
This line doesn't contain anything the filename doesn't already say.

I'd prefer it if it was either removed or you added a link to the project (like that github link!)

Reply:
Done ✔️